Service crushing

Sometimes exporter crushes with Traceback
After restart it works fine. 

```
level=info ts=2021-03-19T10:25:42.792620942Z caller=main.go:74 msg="starting digitalocean_exporter" version=0.6.1 revision=60ab0b6a7c5fac17f003e1df01a0c45639682945 buildDate=2020-09-29T10:18:22+0000 goVersion=go1.15.2

level=info ts=2021-03-19T10:25:42.7934958Z caller=main.go:124 msg=listening addr=:9212

level=warn ts=2021-03-19T10:26:01.710181373Z caller=droplet.go:99 msg="can't list droplets" err="Get \"https://api.digitalocean.com/v2/droplets?page=5\": context deadline exceeded"

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x7ed071]
goroutine 180 [running]:
github.com/metalmatze/digitalocean_exporter/collector.(*DropletCollector).Collect(0xc000098300, 0xc000482060)
    /go/src/github.com/metalmatze/digitalocean_exporter/collector/droplet.go:111 +0x1b1
github.com/prometheus/client_golang/prometheus.(*Registry).Gather.func1()
    /go/pkg/mod/github.com/prometheus/client_golang@v1.2.1/prometheus/registry.go:443 +0x1a2
created by github.com/prometheus/client_golang/prometheus.(*Registry).Gather
    /go/pkg/mod/github.com/prometheus/client_golang@v1.2.1/prometheus/registry.go:535 +0xe8e
stream closed
```

--- a/collector/droplet.go
+++ b/collector/droplet.go
@@ -100,6 +100,7 @@ func (c *DropletCollector) Collect(ch chan<- prometheus.Metric) {
 				"msg", "can't list droplets",
 				"err", err,
 			)
+			return
 		}
 
 		// append the current page's droplets to our list

